addons/l10n_il/i18n/he_IL.po

```diff
@@ -4,24 +4,34 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.5alpha1+e\n"
+"Project-Id-Version: Odoo Server 15.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-09-02 17:16+0000\n"
-"PO-Revision-Date: 2021-09-02 17:16+0000\n"
+"POT-Creation-Date: 2022-08-31 20:25+0000\n"
+"PO-Revision-Date: 2022-08-31 23:28+0300\n"
 "Last-Translator: \n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: \n"
+"Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: \n"
+"Language: he_IL\n"
+"X-Generator: Poedit 2.3\n"
 
 #. module: l10n_il
+#: model:account.tax,description:l10n_il.1_il_vat_pa_purchase_16
 #: model:account.tax,description:l10n_il.2_il_vat_pa_purchase_16
 #: model:account.tax.template,description:l10n_il.il_vat_pa_purchase_16
 msgid "16%"
-msgstr "מע\"מ 16% (רשות פלסטינית)\t"
+msgstr ""
 
 #. module: l10n_il
+#: model:account.tax,description:l10n_il.1_il_vat_inputs_17
```

Comment:
All these 1_il 2_il something are not needed.  Ideally, you just export the pot/po when the coa has not been installed yet in the db.

Reply:
@jco-odoo 
This is what I get when I export the PO from new DB.

Reply:
@hedshefer It might be good to try it by installing another localization first, so the coa is not installed in the db, but the module is still there.

Reply:
@jco-odoo 
I don't understand. Some of the source terms are coming directly from l10n_il module, so in order to have the correct po it has to be installed, is't?

Reply:
@jco-odoo 
Done